src/kubeChannel.ts

```diff
@@ -0,0 +1,13 @@
+import * as vscode from "vscode";
+
+export const kubeChannel = {
+    channel: vscode.window.createOutputChannel("Kubernetes"),
+
+    showOutput(message: any, title?: string) {
+        if (title) {
+            this.channel.appendLine(`[${title} ${(new Date()).toISOString().replace(/z|t/gi, ' ').trim()}]`);

```

Comment:
Can we extract a named variable here to make the transform more self-documenting?

Reply:
nice.